master/internal/db/postgres_checkpoints_intg_test.go

```diff
@@ -60,7 +60,7 @@ func TestUpdateCheckpointSize(t *testing.T) {
 				ckpt := uuid.New()
 				checkpointIDs = append(checkpointIDs, ckpt)
 				// Ensure it works with both checkpoint versions.
-				if i == 0 && j == 0 && k == 0 {
```

Comment:
change here is that the new query won't work on trials that have both a v1 and v2 version.

Though I think this might be impossible or very rare?